bin/alluxio-start.sh

```diff
@@ -187,8 +187,7 @@ start_job_worker() {
   local nworkers=${ALLUXIO_JOB_WORKER_COUNT:-1}
   for (( c = 1; c < ${nworkers}; c++ )); do
     echo "Starting job worker #$((c+1)) @ $(hostname -f). Logging to ${ALLUXIO_LOGS_DIR}"
-    (ALLUXIO_JOB_WORKER_JAVA_OPTS=${ALLUXIO_JOB_WORKER_JAVA_OPTS} \
-    nohup ${BIN}/launch-process job_worker > ${ALLUXIO_LOGS_DIR}/job_worker.out 2>&1) &
+    ( nohup ${BIN}/launch-process job_worker > ${ALLUXIO_LOGS_DIR}/job_worker.out 2>&1) &
```

Comment:
This is still required because above we set `-Dalluxio.job.worker.rpc.port=0 ....`. However, this is only used if `ALLUXIO_JOB_WORKER_COUNT` > 1. I don't think this is documented anywhere anyways. I would just revert